### PR TITLE
updated sol gating and rebased with main

### PR DESCRIPTION
**Pull request by hadiamjad**

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -1 +1 @@
-18.17.0
+18.17.0
```
Comment (mrose17) [inline .node-version:1]: Can we get the newline put back?

File: package.json [hunk 2]
```diff
@@ -43,16 +43,17 @@
     "pretty-quick": "3.1.3",
     "style-loader": "3.3.3",
     "svgo-loader": "4.0.0",
+    "ts-jest": "29.1.1",
```
Comment (mrose17) [inline package.json:46]: can we re-order the dependencies so that only the new one `buffer` shows up in the diff?

File: public/branding-config-dev.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
-  "backgroundColor": "#000000",
+  "X-backgroundColor": "#000000",
```
Comment (mrose17) [inline public/branding-config-dev.json:2]: can we get rid of the two changes in this file? I'm not sure how they keep showing up.

File: public/branding-config-dev.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
   "didPageUrl": "",
   "inviteDomain": "https://talk.brave.software",
   "logoClickUrl": "",
   "logoImageUrl": "https://talk.brave.software/images/brave_logo_dev.svg",
-  "premeetingBackground": "url(https://talk.brave.software/images/background.svg) no-repeat center fixed",
+  "X-premeetingBackground": "url(https://talk.brave.software/images/background.svg) no-repeat center fixed",
```
Comment (mrose17) [inline public/branding-config-dev.json:7]: ditto

**Review by mrose17 — APPROVED** [review]
LGTM - asking @tackley to take a quick look...

File: src/components/WelcomeScreen.tsx [hunk 5]
```diff
@@ -74,13 +88,107 @@ export const WelcomeScreen: React.FC<Props> = ({
...
+
+      const logoStyle: React.CSSProperties = {
+        width: "50px",
+        height: "50px",
+        marginBottom: "10px",
+        borderRadius: "50%",
+      };
+
+      return (
+        <div style={popupContainerStyle}>
```
Comment (tackley) [inline src/components/WelcomeScreen.tsx:145]: Please use the `css` prop provided by @emotion/react rather than the raw `style` prop. Why? Because emotion creates a css class and applies that rather than applying the styles directly to the element.

In this case it doesn't make much difference, other than for consistency, but in other cases especially for repeated elements, this can impact rendering time and page size significantly.

File: src/components/web3/PermissionTypeSelector.tsx [hunk 6]
```diff
@@ -31,17 +32,26 @@ const styles = {
...
   const { t } = useTranslation();
   return (
     <div css={{ display: "flex" }}>
       <button
         onClick={() => setPermissionType("POAP")}
-        css={[styles.base, permissionType === "POAP" && styles.selected]}
+        css={[
+          styles.base,
+          permissionType === "POAP" && styles.selected,
+          web3Account === "SOL" ? disabledButton : null,
```
Comment (tackley) [inline src/components/web3/PermissionTypeSelector.tsx:53]: nit: two different ways of optionally including styles right next to each other is a bit jarring.

I have no real preference between `condition && style` or `condition ? style : null` but please use the same syntax especially when the conditions are right next to each other. Otherwise the next reader of this code has to look twice to understand why these are differently phrased.

File: src/components/web3/SolLogin.tsx [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  onAddressSelected,
+}) => {
+  const [notice, setNotice] = useState<JSX.Element | undefined>();
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    try {
+      window.braveSolana?.on("!!! accountChanged", () => setNotice(undefined));
+    } catch {
+      console.warn("!!! Brave Wallet does not exists");
```
Comment (tackley) [inline src/components/web3/SolLogin.tsx:22]: nit: 

```suggestion
      console.warn("!!! Brave Wallet does not exist");
```

File: src/components/web3/SolLogin.tsx [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+      setNotice(<span>{t("wallet_connect_request")}</span>);
+      web3LoginSol()
+        .then((address) => {
+          onAddressSelected(address, "login");
+          setNotice(undefined);
+        })
+        .catch((err) => {
+          console.error("!!! web3 window.braveSolana.connect", err);
+          setNotice(
+            <Trans i18nKey="wallet_connect_failed">
```
Comment (tackley) [inline src/components/web3/SolLogin.tsx:45]: I'm not seeing this key added to `translations.json` below - is this correct?

File: src/jitsi/options.ts [hunk 9]
```diff
@@ -182,6 +182,7 @@ export const jitsiOptions = (
   };
 
   const features = jwt_decode(jwt)?.context?.features;
+  features.recording = "true";
```
Comment (tackley) [inline src/jitsi/options.ts:185]: Is this change intentional? It doesn't seem related to adding sol gating?

**Comment by hadiamjad** [comment]
@tackley  I've fixed ur suggested changes
